Source/NujasDebug/Private/AIStatCollectionActor.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018-2019 The Janus Project | 2034 Complex LLC. All Rights Reserved.
+
+#include "AIStatCollectionActor.h"
+
+// Sets default values
+AAIStatCollectionActor::AAIStatCollectionActor()
+{
+    // Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
+    PrimaryActorTick.bCanEverTick = true;
+	IConsoleManager::Get().RegisterConsoleVariable( 
+		TEXT("nujas.AI"),
```

Comment:
Move this into a static variable to avoid magic string (i,e, on line 31). I would make it a const string. Also I would recommend using `nujas.ai` instead, make it faster to type on console lol.

Reply:
I tried to use a static variable for it, but it actually wouldn't end up registering the variable :/

Reply:
Try with just a string CV_NUJAS_AI = "nujas.AI" and pass that into TEXT(CV_NUJAS_AI) and see if that works?

Reply:
Also I think you should register console var outside of onbegin play to avoid multiple instance. We don't want to register it multiple time: 

https://github.com/ProjectBorealis/PBCharacterMovement/blob/master/Source/PBCharacterMovement/Private/Character/PBPlayerMovement.cpp#L17-L18